lib/generators/manageiq/plugin/plugin_generator.rb

```diff
@@ -105,6 +126,26 @@ def plugin_description
       @plugin_description ||= "#{plugin_short_name.titleize} plugin for #{Vmdb::Appliance.PRODUCT_NAME}."
     end
 
+    def node_version
+      @node_version ||= package_json.dig("engines", "node")
+    end
+
+    def npm_version
+      @npm_version ||= package_json.dig("engines", "npm")
+    end
+
+    def yarn_version
+      @yarn_version ||= package_json.dig("engines", "yarn") || ">= 0.20.1"
```

Comment:
I had to add the ">= 0.20.1" because the ui-classic package.json does not have a "yarn" version in engines, https://github.com/ManageIQ/manageiq-ui-classic/blob/master/package.json#L174-L177

Reply:
This should really be `">= 3.0.0"`, and inserted in ui-classic as well as the plugins.  Even so, the engines section is just informational since we don't use engines-strict, so not a huge deal.